src/data/finset/lattice.lean

```diff
@@ -8,7 +8,8 @@ import data.finset.option
 import data.finset.prod
 import data.multiset.lattice
 import order.complete_lattice
-
+import order.order_dual
```

Comment:
```suggestion
```
It looks like this is already imported transitively, so I don't see a reason to add it